pipeline_kafka.c

```diff
@@ -1176,8 +1214,55 @@ kafka_consume_main(Datum arg)
 	if (broker_version)
 		rd_kafka_conf_set(conf, "broker.version.fallback", broker_version, NULL, 0);
 
-	kafka = rd_kafka_new(RD_KAFKA_CONSUMER, conf, err_msg, sizeof(err_msg));
-	rd_kafka_set_logger(kafka, consumer_logger);
+	if (consumer.group_id)
+	{
+		rd_kafka_conf_set(conf, "group.id", consumer.group_id, NULL, 0);
+		rd_kafka_conf_set(conf, "offset.store.method", "broker", NULL, 0);
+		rd_kafka_topic_conf_set(topic_conf, "topic.auto.offset.reset", "latest", NULL, 0);
```

Comment:
This parameter is only used when there are no existing offsets stored in the broker for the given group. I see what you're saying though, which is that we should respect the offset parameter even when reading from broker-based offsets. I'll add that.